src/platform/utilities/feature-toggles/index.jsx

```diff
@@ -11,29 +14,79 @@ function makeEnvironmentToggleValues(env = environments) {
   };
 }
 
+function getBootstrappedToggleValues() {
+  return {
+    facilityLocatorShowCommunityCares: true,
+  };
+}
+
+const initialToggleValues = {
+  ...makeEnvironmentToggleValues(),
+  ...getBootstrappedToggleValues(),
+};
+
+let currentToggleValues = initialToggleValues;
+const getToggleValues = () => currentToggleValues;
+
+async function connectFeatureToggle(
+  dispatch,
+  toggleValues = currentToggleValues,
+) {
+  dispatch({
+    type: 'FETCH_TOGGLE_VALUES_STARTED',
+    payload: toggleValues,
+  });
+
+  if (!environments.isProduction) {
+    const newToggleValues = await fetchToggleValues();
+
+    dispatch({
+      type: 'FETCH_TOGGLE_VALUES_SUCCEEDED',
+      payload: newToggleValues,
+    });
+  }
+}
+
+addSubscriberCallback(newToggleValues => {
+  currentToggleValues = {
+    ...currentToggleValues,
+    ...newToggleValues,
+  };
+});
+
+const subscribeToToggleUpdates = callback => addSubscriberCallback(callback);
+
 const ToggleContext = React.createContext();
 
 function ToggleProvider(props) {
```

Comment:
I tried using the context API for wrapping the CC feature but the feature its display is driven by a list of constants that build a `<ul />`. I tried using this at first which is why there are updates to the effect.